video/out/vo.c

```diff
@@ -75,10 +75,8 @@ static const struct vo_driver *const video_out_drivers[] =
 #if HAVE_ANDROID
     &video_out_mediacodec_embed,
 #endif
-    &video_out_gpu,
-#if HAVE_LIBPLACEBO
     &video_out_gpu_next,
-#endif
+    &video_out_gpu,
```

Comment:
@llyyr I think @haasn wanted to keep it non-default for now. If we change it, we can merge wrap, I guess.

Reply:
Sorry, do you mean drop the commit that changes the probeorder or drop every commit besides the one that makes libplacebo a required dependency?

I could also make it like lua, enabled by default unless the user sets -Dlibplacebo=false